rhizome/host/bin/prep_host.rb

```diff
@@ -28,6 +28,13 @@
   safe_write_to_file("/root/.bashrc", "#{bashrc_content}\n PS1='#{colored_prompt_code}'")
 end
 
+r "apt-get -y install figlet"
```

Comment:
Unless you think we have a lot of dynamic text to be worth adding delay to "prep" to install figlet, I suggest running figlet and saving the resulting output.

Also, can we get an environment-sensitive color on the MOTD? similar code to your prompt code